### PR TITLE
Fix Cannot read properties of undefined (reading 'startup')

### PR DESCRIPTION
**Pull request by breakone**

### CODE REVIEW

**Comment by bluvulture** [comment]
hey @breakone 

Could you plase sign the CLA and re-trigger the check?

Tnx

**Comment by breakone** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by breakone** [comment]
recheck

**Comment by breakone** [comment]
recheck

**Comment by breakone** [comment]
@bluvulture I signed the CLA and tried to retrigger the CLA bot (recheck comment)

File: public/js/document/newsletters/sendingPanel.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ pimcore.bundle.newsletter.document.newsletters.sendingPanel = Class.create({
             for(var i = 0; i < adapterNames.length; i++) {
                 //skip adapters that require a namespace that is not available
                 let requiredNamespace = pimcore.bundle.newsletter.document.newsletters.addressSourceAdapters[adapterNames[i]].prototype.requiredNamespace;
-                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace]['startup'] === 'undefined') {
+                if (requiredNamespace && (typeof pimcore.bundle[requiredNamespace] === 'undefined' || typeof pimcore.bundle[requiredNamespace]['startup'] === 'undefined')) {
```
Comment (kingjia90) [inline public/js/document/newsletters/sendingPanel.js]:
```suggestion
                if (requiredNamespace && typeof pimcore.bundle[requiredNamespace]?.startup === 'undefined') {
```
should be easier to read

**Comment by kingjia90** [comment]
The base branch should be `1.1`, could you please rebase it?

**Pull request merged**